akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/scaladsl/ClusterShardingSpec.scala

```diff
@@ -337,6 +332,27 @@ class ClusterShardingSpec
       exc.getMessage should include("[10 ms]") // timeout
     }
 
+    "EntityRef - logs extra context if ask Future fails with exception" in {
+      val ignorantKey = EntityTypeKey[TestProtocol]("ignorant")
+
+      sharding.init(Entity(ignorantKey)(_ => Behaviors.ignore[TestProtocol]).withStopMessage(StopPlz()))
+
+      val ref = sharding.entityRefFor(ignorantKey, "sloppy")
+
+      val exc =
+        LoggingTestKit
+        .error("akka.pattern.AskTimeoutException when sending")
+        .withMessageContains("to entity identified by [slopp]")
```

Comment:
The reason why I'm leaving the command out here is that this test is using an object and strange enough the logged command will be `ClusterShardingSpec$WhoAreYou`, but the assertion text will be `ClusterShardingSpec$WhoAreYou$`. 

Not clear how this happens, probably because of some conversion down the road. In any case, I didn't find it was worth spent time to find it out. The assertion as it is is already providing enough check that we have the logging.